helm/cluster-azure/values.schema.json

```diff
@@ -524,10 +524,15 @@
                             "description": "Name of the image definition in the Gallery",
                             "default": ""
                         },
+                        "variant": {
+                            "type": "string",
+                            "title": "Image variant",
+                            "default": "1"
+                        },
                         "version": {
                             "type": "string",
                             "title": "Image version",
-                            "default": "3510.2.5"
+                            "default": "3815.2.0"
```

Comment:
@erkanerol What's `image.version`, is it Flatcar version?

I see, this version is also changed, when I built new image.
```
"version": {
                            "type": "string",
                            "title": "Image version",
                            "default": "3815.2.0"
                        }
```

If this version changes for `v1.25.16` release, then we don't need variant hack.

Reply:
Yes. This refers to Flatcar version. I believe we need the variant in general since we may want to recreate an image with the same k8s and flatcar version. Did you add the variant to 1.25 image manually? I guess we should add it to image-builder and increase the variant incrementally. Do you know how CAPA does it?

Reply:
> Did you add the variant to 1.25 image manually?
No, it's added by capi-image-builder, we just update the `SUFFIX` to add variant and manually trigger the image build. By default, `SUFFIX` is set to `gs`.

CAPA also did the same, manually triggered the image by updating SUFFIX.
- https://github.com/giantswarm/cluster-aws/pull/525